test/evergreen.yml

```diff
@@ -121,6 +121,8 @@ functions:
           ${HAVE_UNITTEST} \
           ${NON_BARRIER_DIAGNOSTIC_YIELDS|} \
           ${HAVE_DIAGNOSTIC|} \
+          ${CLANG_C_VERSION|} \
```

Comment:
We're still having some issues with the CLANG_C(XX)_VERSION flags. It's being tracked in WT-11239 so we can drop them from this PR and handle it there.

Reply:
I'm trying to remove some of it. I don't know if 6.0 works with us now.

Reply:
Can't resolve. Will remove.